src/kv/msg_pack_serialise.h

```diff
@@ -2,10 +2,9 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
-#include "../ds/msgpack_adaptor_nlohmann.h"
-#include "../ds/serialized.h"
+#include "ds/msgpack_adaptor_nlohmann.h"
```

Comment:
It seems all of our relative includes are unnecessary - I may do a pass removing them for consistency and clarity. They should all be relative to `src/`, and eventually they should get a `ccf/` prefix.

Reply:
I don't know if you're referring to those too but I quite like the files in `test/` to use relative includes for the class they test. Maybe it's just me...

Reply:
I don't have a strong opinion, but it seems cleaner to use the same name for a file everywhere we can? So we can replace `"ds/foo.h"` and not worry about missing `"../../../../foo.h"`.

This also makes it easier to move the test files around, or build them externally, where they're testing the external API that 'real' things use. The exception might be internal or test-only headers that aren't visible externally, but I think these would just be under `_private` or `test` rather than relative?

Reply:
Yeah, consistency certainly is more important. In any case, I'm not too fussed either way.